mksusecd

```diff
@@ -538,6 +554,17 @@ Create ISO image:
                                 instead. MODULE_LIST may be space or comma separated.
                                 Note: this option takes a variable number of arguments. So
                                 it may be necessary to terminate the arg list with '--'.
+      --addon RPM_LIST          A list of RPMs that should be made available as an add-on to
+                                the main product.
+                                Note: this option takes a variable number of arguments. So
+                                it may be necessary to terminate the arg list with '--'.
+      --addon-name NAME         Use NAME as the add-on name.
+                                If unset, the auto-generated name 'Add-On NUM' is used, with NUM
+                                set to the smallest number that avoids name conflicts.
+      --addon-alias ALIAS       Set repo alias to ALIAS.
+                                If unset, an alias based on the repo name is generated.
+      --addon-prio NUM          Set add-on repository priority to NUM; lower NUM means higher
+                                priority (default: 60).
```

Comment:
Maybe this is too long but I'd add something like this to the description:

> The default installation repositories have priority 99. Any smaller number for the add-on repo will prefer the add-on packages even though the package version number is smaller than in the standard repository.

Or put it into a documentation...